docs/02.API-REFERENCE.md

```diff
@@ -4736,6 +4736,77 @@ main (void)
 - [jerry_init](#jerry_init)
 - [jerry_cleanup](#jerry_cleanup)
 - [jerry_exec_snapshot](#jerry_exec_snapshot)
+- [jerry_parse_and_save_static_snapshot](#jerry_parse_and_save_static_snapshot)
+
+
+## jerry_parse_and_save_static_snapshot
+
+**Summary**
+
+Generate static snapshot from the specified source code.
+
+Unlike normal snaphots static snaphots are fully executed from ROM. Not
+even their header is loaded into the RAM. However they can only depend
+on magic strings and 28 bit integer numbers. Regular expression literals
+are not supported as well.
+
+**Prototype**
+
+```c
+size_t
+jerry_parse_and_save_static_snapshot (const jerry_char_t *source_p,
+                                      size_t source_size,
+                                      bool is_for_global,
+                                      bool is_strict,
+                                      uint32_t *buffer_p,
+                                      size_t buffer_size);
+```
+
+- `source_p` - script source, it must be a valid utf8 string.
+- `source_size` - script source size, in bytes.
+- `is_for_global` - snapshot would be executed as global (true) or eval (false).
+- `is_strict` - strict mode
+- `buffer_p` - buffer to save snapshot to.
+- `buffer_size` - the buffer's size.
```

Comment:
What does the size mean here? Byte size? or element size?

Reply:
Byte size just like the other cases (this is mostly a copy-paste). And the plan is rewriting the saving API anyway.